dash/examples/ex.06.pattern-block-visualizer/main.cpp

```diff
@@ -240,11 +301,16 @@ cli_params parse_args(int argc, char * argv[])
     } else if (flag == "-p") {
       params.cout    = true;
       i -= 2;
+    } else {
+      print_usage(argv);
+      exit(EXIT_FAILURE);
     }
   }
+
   return params;
 }
 
+static
```

Comment:
Just wondering: why the statics? These are functions in main.cpp, I don't see how external linkage would hurt?